test/integration/targets/user/tasks/test_create_user.yml

```diff
@@ -55,13 +55,25 @@
 - name: validate changed results for testcase 1
   assert:
     that:
-      - "user_test1.results[0] is not changed"
-      - "user_test1.results[1] is not changed"
-      - "user_test1.results[2] is not changed"
-      - "user_test1.results[3] is not changed"
-      - "user_test1.results[4] is not changed"
+      - 'user_test1.results[0] is not changed'
+      - 'user_test1.results[1] is not changed'
+      - 'user_test1.results[2] is not changed'
+      - 'user_test1.results[3] is not changed'
+      - 'user_test1.results[4] is not changed'
       - "user_test1.results[0]['state'] == 'present'"
       - "user_test1.results[1]['state'] == 'present'"
       - "user_test1.results[2]['state'] == 'present'"
       - "user_test1.results[3]['state'] == 'present'"
       - "user_test1.results[4]['state'] == 'present'"
+
+- name: register user informations
+  when: ansible_facts.system == 'Darwin'
+  command: dscl . -read /Users/ansibulluser
+  register: user_test3
+
+- name: validate user defaults
+  when: ansible_facts.system == 'Darwin'
+  assert:
+    that:
+      - "'RealName: ansibulluser' in user_test3.stdout_lines "
+      - "'PrimaryGroupID: 20' in user_test3.stdout_lines "
```

Comment:
I'd love some insight about this test I wrote. It does the job but I can't help to think it doesn't feel right. ^^